addons/longhorn/template/testgrid/k8s-ctrd.yaml

```diff
@@ -137,7 +137,8 @@
     validate_testfile rwtest testfile.txt
     validate_read_write_object_store postupgrade upgradefile.txt
 
-- name: upgrade-from-oldest-longhorn
+# can only upgrade 1 minor version at a time
+- name: upgrade-from-oldest-longhorn-1.2.x
```

Comment:
this just duplicates the existing 'latest' test then
I'd prefer for there to be code within the addon that skips the upgrade if it's too large

Reply:
latest will upgrade from 1.3.1 once a new version is released. even so i addressed this